chord_metadata_service/phenopackets/models.py

```diff
@@ -302,145 +201,255 @@ def get_project_id(self) -> Optional[str]:
+    ACTIONABLE = 'ACTIONABLE'
+    THERAPEUTIC_ACTIONABILITY_CHOICES = (
+        (UNKNOWN_ACTIONABILITY, 'Unknown actionability'),
+        (NOT_ACTIONABLE, 'Not actionable'),
+        (ACTIONABLE, 'Actionable'),
+    )
+    acmg_pathogenicity_classification = models.CharField(max_length=200, choices=VARIANT_INTERPRETATION_STATUS,
+                                                         default='NOT_PROVIDED',
+                                                         help_text=rec_help(d.VARIANT_INTERPRETATION,
+                                                                            "acmg_pathogenicity_classification"))
+    therapeutic_actionability = models.CharField(max_length=200, choices=THERAPEUTIC_ACTIONABILITY_CHOICES,
+                                                 default='UNKNOWN_ACTIONABILITY',
+                                                 help_text=rec_help(d.VARIANT_INTERPRETATION,
+                                                                    "therapeutic_actionability"))
+    variation_descriptor = models.ForeignKey(VariationDescriptor, on_delete=models.CASCADE,
+                                             help_text=rec_help(d.VARIANT_INTERPRETATION, "variant"))
+
+    def __str__(self) -> str:
         return str(self.id)
 
 
-#############################################################
-#                                                           #
-#                    Interpretation                         #
-#                                                           #
-#############################################################
-
-
-class GenomicInterpretation(models.Model):
+class GenomicInterpretation(BaseTimeStamp):
     """
     Class to represent a statement about the contribution
     of a genomic element towards the observed phenotype
 
     FHIR: Observation
     """
-
+    UNKNOWN_STATUS = 'UNKNOWN_STATUS'
+    REJECTED = 'REJECTED'
+    CANDIDATE = 'CANDIDATE'
+    CONTRIBUTORY = 'CONTRIBUTORY'
+    CAUSATIVE = 'CAUSATIVE'
     GENOMIC_INTERPRETATION_STATUS = (
-        ('UNKNOWN', 'UNKNOWN'),
-        ('REJECTED', 'REJECTED'),
-        ('CANDIDATE', 'CANDIDATE'),
-        ('CAUSATIVE', 'CAUSATIVE')
-        )
-    status = models.CharField(max_length=200, choices=GENOMIC_INTERPRETATION_STATUS,
-                              help_text='How the call of this GenomicInterpretation was interpreted.')
-    gene = models.ForeignKey(Gene, on_delete=models.CASCADE,
-                             blank=True, null=True, help_text='The gene contributing to the diagnosis.')
-    variant = models.ForeignKey(Variant, on_delete=models.CASCADE,
-                                blank=True, null=True, help_text='The variant contributing to the diagnosis.')
+        (UNKNOWN_STATUS, 'Unknown status'),
+        (REJECTED, 'Rejected'),
+        (CANDIDATE, 'Candidate'),
+        (CONTRIBUTORY, 'Contributory'),
+        (CAUSATIVE, 'Causative')
+    )
+    subject_or_biosample_id = models.CharField(
```

Comment:
in the database, it might be better if this is represented as two foreign keys, one to subject and one to biosample, with a constraint that at least one must be null